custom_components/plugwise/const.py

```diff
@@ -27,6 +27,7 @@
 
 # Binary Sensor constants
 BINARY_SENSORS: Final = "binary_sensors"
+BATTERY_STATE: Final = "low_battery"
```

Comment:
Just wondering if we should just make it BATTERY_LOW or LOW_BAT..?

Reply:
I don't think so, it will only cause confusion.

The name of the binary_sensor in HA should be `battery_state`, that fits best to the values of the device_class.
So let's "hide" the different name in the backend-output as soon as possible = in the constant definition.